src/app/static/src/tests/integration/adr-dataset.itest.ts

```diff
@@ -206,7 +206,7 @@ describe("ADR dataset-related actions", () => {
         expect(commit.mock.calls[1][0]["type"]).toBe(BaselineMutation.PopulationUpdated);
         expect(commit.mock.calls[1][0]["payload"]["filename"])
             .toBe("population.csv");
-    }, 7000);
+    }, 10000);
```

Comment:
Increased to 10k to avoid intermittent failure. I think ADR server has become slower.